source/kaleidic/sil/plugin/imap/register.d

```diff
@@ -151,50 +152,47 @@ Size terms, where size is the entire message size in bytes.
     larger(size), smaller(size).`);
 
         // Flags.
-        handlers.registerHandler!(() => new SearchExpr(FlagTerm(FlagTerm.Flag.Answered)))("answered", termDoc);
-        handlers.registerHandler!(() => new SearchExpr(FlagTerm(FlagTerm.Flag.Deleted)))("deleted", termDoc);
-        handlers.registerHandler!(() => new SearchExpr(FlagTerm(FlagTerm.Flag.Draft)))("draft", termDoc);
-        handlers.registerHandler!(() => new SearchExpr(FlagTerm(FlagTerm.Flag.Flagged)))("flagged", termDoc);
-        handlers.registerHandler!(() => new SearchExpr(FlagTerm(FlagTerm.Flag.New)))("new", termDoc);
-        handlers.registerHandler!(() => new SearchExpr(FlagTerm(FlagTerm.Flag.Old)))("old", termDoc);
-        handlers.registerHandler!(() => new SearchExpr(FlagTerm(FlagTerm.Flag.Recent)))("recent", termDoc);
-        handlers.registerHandler!(() => new SearchExpr(FlagTerm(FlagTerm.Flag.Seen)))("seen", termDoc);
-        handlers.registerHandler!(() => new
-                                  SearchExpr(FlagTerm(FlagTerm.Flag.Unanswered)))("unanswered", termDoc);
-        handlers.registerHandler!(() => new
-                                  SearchExpr(FlagTerm(FlagTerm.Flag.Undeleted)))("undeleted", termDoc);
-        handlers.registerHandler!(() => new SearchExpr(FlagTerm(FlagTerm.Flag.Undraft)))("undraft", termDoc);
-        handlers.registerHandler!(() => new
-                                  SearchExpr(FlagTerm(FlagTerm.Flag.Unflagged)))("unflagged", termDoc);
-        handlers.registerHandler!(() => new SearchExpr(FlagTerm(FlagTerm.Flag.Unseen)))("unseen", termDoc);
+        handlers.registerHandler!(() => cast(const) new SearchExpr(FlagTerm(FlagTerm.Flag.Answered)))("answered", termDoc);
```

Comment:
`cast(const) new SearchExpr` it better to replace with `new const SearchExpr`. Looks nicer and the last one expression returns a mutable pointer to a const structure, which potentially is a more friendly API for the SIL Variable.

Reply:
Hey @otrho, thank you for finishing the PR. Just one nitpick above, other changes LGTM.

Reply:
Argh, I thought I tried `new const SearchExpr` and it failed... will try again, patch and redo.  Thanks.  I've only started writing in D since I started at Symmetry so little things like this are new to me. :smiley: